### PR TITLE
Gatsby theme blog 2.0

### PR DESCRIPTION
**Pull request by laurieontech**
This is an umbrella PR for changes coming in `gatsby-theme-blog`. This will include breaking changes that result in a 2.0 version of the theme. 

This branch and PR is currently a WIP.

### CODE REVIEW

File: packages/gatsby-theme-blog/src/components/seo.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ import PropTypes from "prop-types"
 import Helmet from "react-helmet"
 import { useStaticQuery, graphql } from "gatsby"
 
-function SEO({ description, lang, meta, keywords, title }) {
+function SEO({ description, lang, meta, keywords, title, imageSource }) {
```
Comment (therockstorm) [inline packages/gatsby-theme-blog/src/components/seo.js]: Should this be added to `SEO.propTypes` below?

Comment (laurieontech) [reply packages/gatsby-theme-blog/src/components/seo.js]: It should indeed. At the moment this branch is based on existing open PRs. So I've gone ahead and made this change in #23107.

Comment (laurieontech) [reply packages/gatsby-theme-blog/src/components/seo.js]: It should indeed. At the moment this branch is based on existing open PRs. So I've gone ahead and made this change in #23107.

**Comment by LekoArts** [comment]
@laurieontech Can this be closed as you work on this in the separate repo?

**Comment by laurieontech** [comment]
Absolutely. Thought I already had. That must have been 1.6 :)

**Pull request closed without merging**